Source/JavaScriptCore/runtime/JSGenericTypedArrayViewConstructor.cpp

```diff
@@ -111,24 +215,18 @@ JSC_DEFINE_HOST_FUNCTION(uint8ArrayConstructorFromHex, (JSGlobalObject* globalOb
     RETURN_IF_EXCEPTION(scope, { });
 
     size_t count = static_cast<size_t>(view.length() / 2);
-    for (size_t i = 0; i < count * 2; ++i) {
-        int digit = parseDigit(view[i], 16);
-        if (UNLIKELY(digit == -1))
-            return JSValue::encode(throwSyntaxError(globalObject, scope, "Uint8Array.prototype.fromHex requires a string containing only \"0123456789abcdefABCDEF\""_s));
-    }
-
     JSUint8Array* uint8Array = JSUint8Array::create(globalObject, globalObject->typedArrayStructure(TypeUint8, false), count);
```

Comment:
Since the memory will be fully written, couldn't uninitialized memory be used?

Reply:
Done.